### PR TITLE
Remove StatusBoard "fullname" search parameter.

### PR DESCRIPTION
**Pull request by tinaafitz**
Removed the Status Board "Fullname" search parameter for [OSDEV-1296]https://issues.redhat.com/browse/OSDEV-1296l).  ocm-sdk-go: https://github.com/openshift-online/ocm-sdk-go/pull/875

### CODE REVIEW

**Comment by smcavey** [comment]
lgtm

**Comment by tinaafitz** [comment]
Hi @smcavey, Please review. :-)

File: CHANGES.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 # Changes
 
 This document describes the relevant changes between releases of the API model.
+## 0.0.343 December 12 2023
```
Comment (smcavey) [inline CHANGES.md:4]: Should be 0.0.352 now and date should be updated

File: CHANGES.md [hunk 2]
```diff
@@ -1,6 +1,8 @@
...
+- Remove the `Fullname` search parameter.
```
Comment (smcavey) [inline CHANGES.md:5]: I'd add "from `status_board` model" or something to be more specific

Comment (tinaafitz) [reply CHANGES.md]: Thank you. :-) Updated.

File: CHANGES.md (hunk 1 above)
Comment (tinaafitz) [reply CHANGES.md]: Thank you. :-) Updated.

**Comment by smcavey** [comment]
lgtm :)

**Review by tzvatot — CHANGES_REQUESTED** [review]
The release will need to go as a seperated MR.

**Comment by tinaafitz** [comment]
Thank you so much, @tzvatot. I removed the release information.

**Review by tzvatot — APPROVED** [review]

**Comment by tzvatot** [comment]
LGTM

**Pull request merged**